.github/workflows/release.yml

```diff
@@ -14,13 +14,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - uses: actions/setup-go@v2
+      - uses: actions/setup-go@v4
         with:
-          go-version: '1.19'
+          go-version: '1.20.3'
```

Comment:
We learned in the relayer that `'1.20'` for `actions/setup-go` will actually use go 1.20 (the first) instead of the latest (since go does not use `1.20.0` for the first 1.20 release for some reason).

Checking their docs now though, it looks like we can use `^1.20.0` to use the latest 1.20.x